src/winadinf.c

```diff
@@ -1428,6 +1428,8 @@ static const char *get_phys_address (const void *a, ULONG len, BOOL show_manuf)
   char        *p, *p_max = work_buf + sizeof(work_buf) - 4;
   ULONG        i;
 
+  ARGSUSED (show_munuf)
```

Comment:
Should be `show_manuf` and a trailing `;`. Fixed in: https://github.com/gvanem/Watt-32/commit/059ee184e49149d7ec30ce5a26ea0c1226910038.